pages/common/install.md

```diff
@@ -0,0 +1,24 @@
+# install
+
+> Copy files and set attributes.
```

Comment:
It would be really useful to explain why the tool is called "install". See for example the description of the kill command.